history/taskana-simplehistory-rest-spring/src/main/resources/sql.sample-data/clear-db.sql

```diff
@@ -1,3 +1,3 @@
 -- the order is important!
```

Comment:
Do we still use these SQL files?

Reply:
yes, they get used in the taskana-simplehistory-rest-spring tests, but probably could be replaced once we get the test dependencies module with the test data from our SampleDataGenerator